lib/jobFactory.js

```diff
@@ -55,8 +55,11 @@ class JobFactory extends BaseFactory {
      */
     create(config) {
         const c = config;
+        const jobDisabledByDefault = convertToBool(
+            getAnnotations(c.permutations[0], 'screwdriver.cd/jobDisabledByDefault')
+        );
 
-        c.state = 'ENABLED';
+        c.state = jobDisabledByDefault === true ? 'DISABLED' : 'ENABLED';
```

Comment:
should we call it `enabled` instead and by default it will be true, users can manually set it to false?

Reply:
Either option looks good. I believe what really matters is the default value based on how we name them.

Reply:
Talked with @pritamstyz4ever  offline and we agreed to leave it as is for now
open to changes in the future if we prefer `jobEnabledByDefault`, `defaultJobState`, etc